scripts/docker/init.bash

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+for dir in /built-binaries/*; do
+  envVar="$(echo "${dir}" | cut -d "/" -f 3 | tr '[:lower:]' '[:upper:]' | tr '-' '_')_BINARY"
+  eval "export ${envVar}=$dir/run";
+done
+
+pushd /repo
+. "/ci/diego-release/helpers/configure-binaries.bash"
+. "/ci/shared/helpers/helpers.bash"
+expand_functions
+
+configure_db "${DB}"
```

Comment:
@geofffranks I think this all happens [as part of running the test.](https://github.com/cloudfoundry/wg-app-platform-runtime-ci/blob/main/shared/tasks/run-bin-test/task.bash#L37-L48). Were you seeing something other than that?